skypy/galaxies/morphology.py

```diff
@@ -251,6 +251,11 @@ def ryden04_ellipticity(mu_gamma, sigma_gamma, mu, sigma, size=None):
     size : int or tuple of ints or None
         Size of the sample. If `None` the size is inferred from the parameters.
 
+    Returns
+    -------
+    ellipticity: (size,) array_like
+        Ellipticity distribution sampled from the Ryden 2004 model [1].
```

Comment:
```suggestion
        Ellipticities sampled from the Ryden 2004 model.
```
- Cant use the `[1]` link more than once
- Slightly clearer text